Readme.md

```diff
@@ -253,6 +253,13 @@ $ heroku pg:vacuum-stats
 $ heroku pg:mandelbrot
 ```
 
+```
+$ heroku pg:user-connections
+Credential      Connections
+──────────────  ───────────
+ua7almfsv0d8tq  24
```

Comment:
so there is a consistency question here potentially with `heroku pg:credentials` which lists the default credential as `default` and not the username. I feel we should stay consistent, and list the credential as `default` here too, any thoughts @uhoh-itsmaciek @coreypurcell ?

Reply:
consistency is probably good. Looks like I can see if the user is default by hitting the credentials endpoint.

Reply:
@camillebaldock I changed it to return default if it is the default credential